resources/user_config.js

```diff
@@ -251,6 +251,7 @@ let UserConfig = {
       textElem = document.createElement('div');
       textElem.id = id;
       textElem.classList.add('text');
+      textElem.classList.add('lang-' + lang);
```

Comment:
Do you think it would make sense for the `loadUser` function (the caller of this `addUnlockText` function) to add `.lang-ja` to the top of the page, e.g. the body element? It seems like this is something that it might be nice to depend on for every overlay via a `.lang-ja .text` selector.

Reply:
Sounds great, I'll try to do this!